app/screens/reusable/EmojiSelection/EmojiSelection.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { ImageBackground } from 'react-native';
+
+import {
+  BottomButtonView,
+  ExtendedText,
+  MainHeader,
+} from '../../../components';
+import { IMAGES } from '../../../assets';
+import { useAppSelector } from '../../../hooks';
+import { generalStyles } from '../../../utils/styles';
+import { IEmojiSelectionScreenProps } from './EmojiSelection.props';
+
+export const EmojiSelectionScreen: React.FC<IEmojiSelectionScreenProps> = ({
+  navigation,
+  route,
+}) => {
+  const { onSubmit } = route.params.data;
+  const { t } = useTranslation();
+
+  // TODO: chande to real nickname
+  const parentNickname = useAppSelector(state => state.user.user?.displayName);
+
+  //   const [emoji, setEmoji] = useState<EmojiType>(EmojiType.Normal);
```

Comment:
redundant comment

Reply:
no, this is an example of functionality
later the storage will be changed to users: {parent: ..., child: ...}
and in this place will need to be corrected